### PR TITLE
Fix/flux remote

### PR DESCRIPTION
**Pull request by andre-merzky**

### CODE REVIEW

File: src/radical/utils/flux.py [hunk 1]
```diff
@@ -144,7 +144,10 @@ def _locked_start_service(self,
         if launcher:
             cmd += shlex.split(launcher)
 
-        cmd += ['flux', 'start', 'bash', '-c', 'echo URI:$FLUX_URI && sleep inf']
+        cmd += ['flux', 'start', 'bash', '-c',
+                'echo "HOST:$(hostname) URI:$FLUX_URI" && sleep inf']
+
+        self._log.debug('flux command', cmd)
```
Comment (mtitov) [inline src/radical/utils/flux.py:150]: missing reference to the cmd in the msg -> `'flux command: %s'`

File: src/radical/utils/flux.py [hunk 2]
```diff
@@ -190,6 +199,8 @@ def _locked_start_service(self,
         self._env       = flux_env
         self._proc      = flux_proc
 
+        self._log.debug('flux uri', flux_uri)
```
Comment (mtitov) [inline src/radical/utils/flux.py:202]: `'flux uri: %s'`

**Review by mtitov — APPROVED** [review]
LGTM! couple comments regarding debug messages

File: src/radical/utils/flux.py (hunk 1 above)
Comment (andre-merzky) [reply src/radical/utils/flux.py:150]: thanks!

File: src/radical/utils/flux.py (hunk 2 above)
Comment (andre-merzky) [reply src/radical/utils/flux.py]: Thanks also, both fixed.

**Pull request merged**